src/desktop/config.coffee

```diff
@@ -81,7 +81,7 @@ module.exports =
   MOBILE_MEDIA_QUERY: "only screen and (max-width: 640px)"
   MOBILE_URL: 'https://m.artsy.net'
   NEW_ARTIST_PAGE_THRESHOLD: 10 # all users will get the new page.
-  NEW_AUTH_MODAL: false
+  NEW_AUTH_MODAL: true
```

Comment:
@l2succes is this okay? I updated this config to default to `true` so people don't inadvertently see the old modals locally

Reply:
Yup that's fine, we'll be removing that flag soon when we clean up the legacy /auth app code